samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -0,0 +1,240 @@
+Parameters such as: CMD code, Frequency, Length or Packet Type are encoded within this command.
+
+The following DTM events are polled using the ``dtm_event_get`` function:
+
+* ``PACKET_REPORTING_EVENT``
+* ``TEST_STATUS_EVENT`` [ ``SUCCESS`` | ``FAIL`` ]
+
+.. figure:: /images/bt_dtm_engine.svg
+   :alt: State machine overview of the DTM
+
+Supported PHYs
+==============
+
+The DTM sample supports all four PHYs specified in DTM, but not all devices support all the PHYs.
+
+.. list-table:: Supported PHYs
+   :header-rows: 1
+
+   * - PHY
+     - nRF5340
+   * - LE 1M
+     - Yes
+   * - LE 2M
+     - Yes
+   * - LE Coded S=8
+     - Yes
+   * - LE Coded S=2
+     - Yes
+
+Vendor Specific packet payload
+==============================
+
+The Bluetooth Low Energy 2-wire UART DTM interface standard reserves the Packet Type, also called payload parameter, with binary value ``11`` for a Vendor Specific packet payload.
+
+The DTM command is interpreted as a Vendor Specific one when both the following conditions are met:
+
+* Its CMD field is set to Transmitter Test, binary ``10``.
+* Its PKT field is set to Vendor Specific, binary ``11``.
+
+Vendor specific commands can be divided into different categories as follows:
+
+* If the Length field is set to ``0`` (symbol ``CARRIER_TEST``), an unmodulated carrier is turned on at the channel indicated by Frequency field.
+  It remains turned on until a ``TEST_END`` or ``RESET`` command is issued.
+* If the Length field is set to ``1`` (symbol ``CARRIER_TEST_STUDIO``), this field value is used by the nRFgo Studio to indicate that an unmodulated carrier is turned on at the channel.
+  It remains turned on until a ``TEST_END`` or ``RESET`` command is issued.
+* If the Length field is set ``2`` (symbol ``SET_TX_POWER``), Frequency field sets the TX power in dBm.
+  The valid TX powers are specified in the product specification ranging from -40 to +4.
+  0 dBm is the reset value.
+  Only the 6 least significant bits will fit in the length field.
+  The two most significant bits are calculated by the DTM module.
+  This is possible because the 6 least significant bits of all valid TX powers are unique.
+  The TX power can be modified only when no Transmitter Test or Receiver Test is running.
+* All other values of Frequency and Length field are reserved.
+
+The DTM-to-Serial adaptation layer
+==================================
+
+``main.c`` is an implementation of the UART interface specified in the`Bluetooth Core Specification`_: Vol. 6, Part F, Chap. 3.
+
+The default selection of UART pins is defined in ``zephyr/boards/arm/board_name/board_name.dts``.
+You can change the defaults using the symbols ``tx-pin`` and ``rx-pin`` in the DTS overlay file at the project level.
+
+Requirements
+************
+
+* The following development kit:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set15_start
+   :end-before: set15_end
```

Comment:
This set supports only Application Core from the nRF5340 DK. However, this sample should be run on the Network Core.

I think we need to define a new set with nRF5340 Network Core support only. There is set4 which supports this target but it also includes nRF52 devices. It's a good starting point for this new set definition.

Reply:
fixed